content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+
+The generated client library can be made available in your workstation, or it can also be uploaded to a GitHub library so that it can be made available for others to use. An example of the GitHub repository of a sample Python client library looks like this: 
+
+![](/img/python-open-api-sdk-repo.jpg "GitHub repository of Python Client library generated using OpenAPI generator")
+
+The client library comes with the following:
+
+* The code for assisting the DSCC API calls.
+* Documentation for this DSCC client library that derived from the API spec.
+* Tests codes that can be used to validate the operation of this client library.
+* Examples of every endpoint available in the README.md
+* Required python dependencies (requirements.txt and test-requirements.txt) that are required for using this SDK
+
+For instance, take a look at the README file of this repository which has the following info.
+
+1. Installation instructions
+
+![](/img/readme.png "Python SDK installation instructions")
+
+2. A sample code to get started with 
+
+![](/img/sample.png "Sample code")
+
+3. Documentation list for all endpoints
+
+![](/img/api-endpoints.png "List of endpoints")
+
+4. Documentation list for all models
+
+![](/img/models.png "List of models")
+
+5. Documentation about authorization
+
+![](/img/auth.png "Authorization of API calls")
+
+Lets run a sample code which displays the access types in DSCC. The usage of each endpoint is given under documentation of each endpoint in the form of a sample code. To execute the operations on the DSCC, all that is needed for a user to do is:
+
+1. Provide an authorized token which is generated from the HPE GreenLake as mentioned in the [blog](https://developer.hpe.com/blog/oauth2-for-hpe-greenlake-data-services-cloud-console/)
+2. Install the Python dependencies using the following command.
+
+```shell
+~$ pip install requirements.txt 
+```
+
+Below is the sample of the code to get the information about the associated user's RBAC association. This code will provide list of the capabilities (port.read, volume.create) of that related user who provided the authorization can exercise. To execute this code, please substitute the YOUR_BEARER_TOKEN with the access token generated in your example.
+
+```python
+import time
+import openapi_client
+from openapi_client.api import authz_api
+from openapi_client.model.error_response import ErrorResponse
+from openapi_client.model.access_controls_response import AccessControlsResponse
+from pprint import pprint
+# Defining the host is optional and defaults to https://eu1.data.cloud.hpe.com
+# See configuration.py for a list of all supported configuration parameters.
+configuration = openapi_client.Configuration(
+    host = "https://eu1.data.cloud.hpe.com"
+)
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
```

Comment:
Unfortunately this is generated directly from the API spec, hence I would rather leave it the way it is so that it can be followed by user. I agree, there should be review on the API spec itself for the proper English use.